sql/moz-fx-data-shared-prod/telemetry/fenix_and_firefox_use_counters/view.sql

```diff
@@ -19,4 +19,4 @@ SELECT
   metric,
   rate
 FROM
-  `moz-fx-data-shared-prod.fenix_derived.fenix_use_counters_v1`
```

Comment:
This was recently added. Are these `_v1`'s going to stick around? If the output for the generated files look mostly the same. I don't think there's a reason to bump the version number

Reply:
Totally agree - I am planning to remove the V1 tables in a future release